tensorboard/webapp/plugins/plugins_container.ts

```diff
@@ -49,28 +56,50 @@ const lastLoadedTimeInMs = createSelector(
   template: `
     <plugins-component
       [activePlugin]="activePlugin$ | async"
-      [noEnabledPlugin]="noEnabledPlugin$ | async"
+      [activePluginId]="activePluginId$ | async"
+      [dataLocation]="dataLocation$ | async"
       [lastUpdated]="lastLoadedTimeInMs$ | async"
+      [pluginLoadState]="pluginLoadState$ | async"
     ></plugins-component>
   `,
   styles: ['plugins-component { height: 100%; }'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class PluginsContainer {
-  readonly activePlugin$ = this.store.pipe(select(activePlugin));
-  readonly noEnabledPlugin$ = combineLatest(
-    this.store.select(activePlugin),
+  readonly activePlugin$ = this.store.select(activePlugin);
+  readonly activePluginId$ = this.store.select(getActivePlugin);
+
+  readonly pluginLoadState$ = combineLatest(
+    this.activePlugin$,
+    this.activePluginId$,
```

Comment:
Hmm, I am now a bit confused. `activePlugin` !== `activePluginId`. Also. `activePlugin` is being passed to child component at line 58. 

Am I missing something?

Reply:
Ah, I see why we can't just remove one of these 2 observables, since both are passed into the child component.

I'm still concerned that a reasonable reader will immediately be confused when seeing the names `activePlugin$` and `activePluginId$`.

Can we rename `activePlugin$` to something like `activeKnownPlugin$`, or at least add a comment on L69 explaining how `activePlugin$` differs from `activePluginId$`?